osquery/events/darwin/event_taps.cpp

```diff
@@ -43,39 +44,45 @@ void EventTappingEventPublisher::tearDown() {
 }
 
 void EventTappingEventPublisher::stop() {
-  WriteLock lock(run_loop_mutex_);
+  WriteLock lock(runLoopMutex_);
 
-  if (run_loop_source_ != nullptr) {
-    CFRunLoopRemoveSource(run_loop_, run_loop_source_, kCFRunLoopCommonModes);
-    CFRelease(run_loop_source_);
-    run_loop_source_ = nullptr;
+  if (runLoopSource_ != nullptr) {
+    CFRunLoopRemoveSource(runLoop_, runLoopSource_, kCFRunLoopCommonModes);
+    CFRelease(runLoopSource_);
+    runLoopSource_ = nullptr;
   }
-  if (run_loop_ != nullptr) {
-    CFRunLoopStop(run_loop_);
-    run_loop_ = nullptr;
+
+  if (eventTap_ != nullptr) {
+    CGEventTapEnable(eventTap_, false);
```

Comment:
Should you do this "disable" before removing the source? Not sure, just curious?